tools/depends/target/python27/Makefile

```diff
@@ -45,6 +45,7 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 ifeq ($(OS),ios)
 	cd $(PLATFORM); patch -p1 < ../make-fork-optional.patch
 	cd $(PLATFORM); patch -p1 < ../urllib.patch
+	cd $(PLATFORM); patch -p1 < ../python_register.patch
```

Comment:
This patch is not required to get it build successful so there is no need for it. Also this doesn't change anything for Linux if Kodi is built with system libs.
The correct fix to hide those warnings would be to pass the external library include directories via '-isystem' instead of `-I`

Reply:
Patch was only applied if os is iOS. Doesn't affect any other platform. Was just removing a bunch of warnings that only cluttered the logs from an Xcode build.
Happy to close if not deemed useful

Reply:
Then it's completely wrong, if it doesn't hide the warnings for all depends based builds.
But as already said if you want it fixed, fix it the correct way for all platforms.

Reply:
As far as im aware, it only affects clang used by xcode 10.2. Didnt see a need to change any other platform for a maybe might kinda possible who knows.

'register' storage class specifier is deprecated and incompatible with C++17

Reply:
So osx would be affected too. Android already shows the warnings in jenkins and if you build with clang for linux you also get them.

Reply:
btw, its not a compiled object warning. The two header files are copied to the includes/python27 folder, and any include of python.h through out the kodi codebase, then gets the warning.
I guess a fix could be to force std=c++11, but that was just removed as a flag in https://github.com/xbmc/xbmc/commit/598e999920f1509110d0ee18d4c7db23646672f4

I dunno what you'd want as a proper fix. Sure i could change the patch to apply for every platform, but im imagining thats not what you want, and as you said wouldnt affect system libs.

Happy to just leave as is, and wait for python3 where its a non issue.